### PR TITLE
Team news and add blog index

### PR DESCRIPTION
**Pull request by willzeng**
This adds a blog index and replaces #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -375,6 +377,12 @@ <h1>Unitary Fund</h1>
 
     </p>
 
+    <p class="subtitle"><a name="news">News</a></p>
+
+    <p>
+        <a href="news.html">Team intro</a>
```
Comment (nathanshammah) [inline index.html]: I missed to change this. it should be 
```
<a href="blog.html">Team intro</a>
```

File: index.html [hunk 2]
```diff
@@ -375,6 +377,12 @@ <h1>Unitary Fund</h1>
 
     </p>
 
+    <p class="subtitle"><a name="news">News</a></p>
+
+    <p>
```
Comment (nathanshammah) [inline index.html]: Actually we can cut all this.

File: posts/unitary_labs_intro.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        <section class="light">
+            <br>
+            <br>
+
+            <center>
+                <h1>Introducing the Technical Team: Unitary Labs</h1>
+            </center>
+            <br>
+            <br>
+            14 March, 2020<br>
```
Comment (nathanshammah) [inline posts/unitary_labs_intro.html]: Maybe we could change this to 19 March, 2020 (or actual release date). Also in the main blog directory.

File: blog.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+                most people.
+            </p>
+
+            <p class="subtitle">Our Blog</p>
+
+
+        </section>
+        <section class="light">
+            <ul>
+                <li>[14 March, 2020] <a href="posts/unitary_labs_intro.html">Introducing the Technical Team: Unitary
```
Comment (nathanshammah) [inline blog.html]: [19 March, 2020] (or release date)

File: index.html (hunk 1 above)
Comment (nathanshammah) [reply index.html]: sorry I mean cut this all as we have no news.

Comment (willzeng) [reply index.html]: removed

File: posts/unitary_labs_intro.html (hunk 3 above)
Comment (willzeng) [reply posts/unitary_labs_intro.html]: removed

File: blog.html (hunk 4 above)
Comment (willzeng) [reply blog.html]: changed

**Pull request merged**